change/@fluentui-react-jsx-runtime-d2a505fc-6ffc-4679-aa63-5b29dcc83f7b.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28373&commitId=30c4518e1f20f671cf7bf177d65743e6dbfc29aa&env=prod&clientType=FLUENTUI)


  <details><summary><b>Accordion Converged</b> 21 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Accordion Converged.visibility+focus - High Contrast.normal.chromium.png|527| Changed|
|Accordion Converged.disabled - High Contrast.normal.chromium.png|1259| Changed|
|Accordion Converged.visibility+focus - High Contrast.focus closed.chromium.png|527| Changed|
|Accordion Converged.expandIconPosition  end  - RTL.normal.chromium.png|25| Changed|
|Accordion Converged.expandIconPosition  end .normal.chromium.png|25| Changed|
|Accordion Converged.disabled - Dark Mode.normal.chromium.png|361| Changed|
|Accordion Converged.visibility+focus - RTL.focus opened.chromium.png|449| Changed|
|Accordion Converged.disabled.normal.chromium.png|484| Changed|
|Accordion Converged.visibility+focus.focus closed.chromium.png|454| Changed|
|Accordion Converged.visibility+focus.normal.chromium.png|454| Changed|
|Accordion Converged.visibility+focus - Dark Mode.focus opened.chromium.png|440| Changed|
|Accordion Converged.icon   Icon-  .normal.chromium.png|336| Changed|
|Accordion Converged.visibility+focus - Dark Mode.focus closed.chromium.png|440| Changed|
|Accordion Converged.visibility+focus - High Contrast.focus opened.chromium.png|527| Changed|
|Accordion Converged.icon   Icon-   - RTL.normal.chromium.png|345| Changed|
|Accordion Converged.visibility+focus - RTL.normal.chromium.png|449| Changed|
|Accordion Converged.visibility+focus.focus opened.chromium.png|454| Changed|
|Accordion Converged.visibility+focus - RTL.focus closed.chromium.png|449| Changed|
|Accordion Converged.visibility+focus - Dark Mode.normal.chromium.png|440| Changed|
|Accordion Converged.size - RTL.normal.chromium.png|1091| Changed|
|Accordion Converged.size.normal.chromium.png|1112| Changed|

</details>


  <details><summary><b>Avatar Converged</b> 20 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.basic - High Contrast.normal.chromium.png|835| Changed|
|Avatar Converged.basic - RTL.normal.chromium.png|558| Changed|
|Avatar Converged.basic - Dark Mode.normal.chromium.png|237| Changed|
|Avatar Converged.image-bad-url+icon.normal.chromium.png|107| Changed|
|Avatar Converged.image-bad-url.normal.chromium.png|118| Changed|
|Avatar Converged.basic.normal.chromium.png|558| Changed|
|Avatar Converged.customSize+name+badge.normal.chromium.png|3510| Changed|
|Avatar Converged.customSize+icon+active.normal.chromium.png|3473| Changed|
|Avatar Converged.color - High Contrast.normal.chromium.png|9701| Changed|
|Avatar Converged.color - Dark Mode.normal.chromium.png|5797| Changed|
|Avatar Converged.color.normal.chromium.png|13001| Changed|
|Avatar Converged.color+active - High Contrast.normal.chromium.png|9701| Changed|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|1285| Changed|
|Avatar Converged.color+active.normal.chromium.png|13001| Changed|
|Avatar Converged.color+active - Dark Mode.normal.chromium.png|5797| Changed|
|Avatar Converged.badgeMask.normal.chromium.png|1283| Changed|
|Avatar Converged.size+name.normal.chromium.png|5527| Changed|
|Avatar Converged.size+icon+badge+square.normal.chromium.png|7199| Changed|
|Avatar Converged.size+image+badge.normal.chromium.png|1841| Changed|
|Avatar Converged.size+inactive+badge.normal.chromium.png|574| Changed|

</details>


  <details><summary><b>AvatarGroup Converged</b> 27 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|AvatarGroup Converged.layoutPie-1 - RTL.default.chromium.png|238968| Changed|
|AvatarGroup Converged.layoutPie-2 - Dark Mode.default.chromium.png|237349| Changed|
|AvatarGroup Converged.layoutPie - Dark Mode.default.chromium.png|242831| Changed|
|AvatarGroup Converged.layoutPie-2 - High Contrast.default.chromium.png|237757| Changed|
|AvatarGroup Converged.layoutPie - RTL.default.chromium.png|337313| Changed|
|AvatarGroup Converged.layoutPie-1 - Dark Mode.default.chromium.png|237307| Changed|
|AvatarGroup Converged.basic - Dark Mode.default.chromium.png|224524| Changed|
|AvatarGroup Converged.layoutPie - High Contrast.default.chromium.png|256306| Changed|
|AvatarGroup Converged.layoutPie-1 - High Contrast.default.chromium.png|237692| Changed|
|AvatarGroup Converged.layoutPie-2 - RTL.default.chromium.png|238965| Changed|
|AvatarGroup Converged.layoutPie.default.chromium.png|337248| Changed|
|AvatarGroup Converged.layoutPie-1.default.chromium.png|238968| Changed|
|AvatarGroup Converged.basic - High Contrast.default.chromium.png|267743| Changed|
|AvatarGroup Converged.layoutPie-2.default.chromium.png|238973| Changed|
|AvatarGroup Converged.basic.default.chromium.png|716548| Changed|
|AvatarGroup Converged.layoutStack - Dark Mode.default.chromium.png|231810| Changed|
|AvatarGroup Converged.layoutStack.default.chromium.png|617991| Changed|
|AvatarGroup Converged.overflowIndicator - Dark Mode.default.chromium.png|223428| Changed|
|AvatarGroup Converged.overflowContent.chromium.png|1606193| Changed|
|AvatarGroup Converged.overflowIndicator - High Contrast.default.chromium.png|266820| Changed|
|AvatarGroup Converged.layoutStack - High Contrast.default.chromium.png|265900| Changed|
|AvatarGroup Converged.overflowContent - Dark Mode.chromium.png|1597198| Changed|
|AvatarGroup Converged.overflowIndicator.default.chromium.png|717660| Changed|
|AvatarGroup Converged.overflowContent - High Contrast.chromium.png|1598943| Changed|
|AvatarGroup Converged.overflowContent - Dark Mode.popoverContentOpen.chromium.png|0| Removed|
|AvatarGroup Converged.overflowContent - High Contrast.popoverContentOpen.chromium.png|0| Removed|
|AvatarGroup Converged.overflowContent.popoverContentOpen.chromium.png|0| Removed|

</details>


  <details><summary><b>Checkbox Converged</b> 36 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Checkbox Converged.checked.rest.chromium.png|244560| Changed|
|Checkbox Converged.disabled+checked.default.chromium.png|244760| Changed|
|Checkbox Converged.label-before.default.chromium.png|244679| Changed|
|Checkbox Converged.label-wrapping.default.chromium.png|256551| Changed|
|Checkbox Converged.label-before - RTL.default.chromium.png|244679| Changed|
|Checkbox Converged.circular+checked.default.chromium.png|244480| Changed|
|Checkbox Converged.label-wrapping - RTL.default.chromium.png|256546| Changed|
|Checkbox Converged.large+circular+mixed.default.chromium.png|245020| Changed|
|Checkbox Converged.large+circular+checked.default.chromium.png|244912| Changed|
|Checkbox Converged.disabled.rest.chromium.png|244808| Changed|
|Checkbox Converged.large - RTL.default.chromium.png|245336| Changed|
|Checkbox Converged.large+checked.default.chromium.png|244969| Changed|
|Checkbox Converged.disabled+mixed.default.chromium.png|244762| Changed|
|Checkbox Converged.circular+mixed.default.chromium.png|244565| Changed|
|Checkbox Converged.large+circular.default.chromium.png|245265| Changed|
|Checkbox Converged.circular.default.chromium.png|244749| Changed|
|Checkbox Converged.large+label-wrapping - RTL.default.chromium.png|257730| Changed|
|Checkbox Converged.large+label-wrapping.default.chromium.png|257732| Changed|
|Checkbox Converged.unchecked - RTL.rest.chromium.png|244694| Changed|
|Checkbox Converged.large+mixed.default.chromium.png|245114| Changed|
|Checkbox Converged.no-label.default.chromium.png|244829| Changed|
|Checkbox Converged.unchecked.rest.chromium.png|244694| Changed|
|Checkbox Converged.required.default.chromium.png|244705| Changed|
|Checkbox Converged.large.default.chromium.png|245336| Changed|
|Checkbox Converged.required+label-before.default.chromium.png|244506| Changed|
|Checkbox Converged.mixed.rest.chromium.png|244679| Changed|
|Checkbox Converged.checked.active.chromium.png|0| Removed|
|Checkbox Converged.checked.hover.chromium.png|0| Removed|
|Checkbox Converged.disabled.active.chromium.png|0| Removed|
|Checkbox Converged.disabled.hover.chromium.png|0| Removed|
|Checkbox Converged.mixed.active.chromium.png|0| Removed|
|Checkbox Converged.mixed.hover.chromium.png|0| Removed|
|Checkbox Converged.unchecked - RTL.active.chromium.png|0| Removed|
|Checkbox Converged.unchecked - RTL.hover.chromium.png|0| Removed|
|Checkbox Converged.unchecked.active.chromium.png|0| Removed|
|Checkbox Converged.unchecked.hover.chromium.png|0| Removed|

</details>


  <details><summary><b>Combobox Converged</b> 61 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Combobox Converged.Appearance- underline.default.chromium.png|244614| Changed|
|Combobox Converged.Appearance- outline (default).default.chromium.png|244783| Changed|
|Combobox Converged.Appearance- filled-darker.default.chromium.png|243787| Changed|
|Combobox Converged.Appearance- filled-lighter.default.chromium.png|244099| Changed|
|Combobox Converged.Disabled option.default.chromium.png|272530| Changed|
|Combobox Converged.Disabled with value.default.chromium.png|244910| Changed|
|Combobox Converged.Invalid- filled-lighter.default.chromium.png|243760| Changed|
|Combobox Converged.Disabled.default.chromium.png|244929| Changed|
|Combobox Converged.Invalid- filled-darker.default.chromium.png|243471| Changed|
|Combobox Converged.Invalid- outline.default.chromium.png|244655| Changed|
|Combobox Converged.Open with grouped options.default.chromium.png|316802| Changed|
|Combobox Converged.Invalid- underline.default.chromium.png|244617| Changed|
|Combobox Converged.Option with long content.default.chromium.png|300615| Changed|
|Combobox Converged.Open with inlinePopup.default.chromium.png|272503| Changed|
|Combobox Converged.Open.default.chromium.png|272503| Changed|
|Combobox Converged.With placeholder.default.chromium.png|244727| Changed|
|Combobox Converged.With multiselect selection.default.chromium.png|272163| Changed|
|Combobox Converged.Size- large.default.chromium.png|245483| Changed|
|Combobox Converged.With value.default.chromium.png|244651| Changed|
|Combobox Converged.Size- small.default.chromium.png|242991| Changed|
|Combobox Converged.With selection.default.chromium.png|272484| Changed|
|Combobox Converged.Appearance- filled-darker.focused.chromium.png|0| Removed|
|Combobox Converged.Appearance- filled-darker.hover.chromium.png|0| Removed|
|Combobox Converged.Appearance- filled-lighter.focused.chromium.png|0| Removed|
|Combobox Converged.Appearance- filled-lighter.hover.chromium.png|0| Removed|
|Combobox Converged.Appearance- outline (default).focused.chromium.png|0| Removed|
|Combobox Converged.Appearance- outline (default).hover.chromium.png|0| Removed|
|Combobox Converged.Appearance- underline.focused.chromium.png|0| Removed|
|Combobox Converged.Appearance- underline.hover.chromium.png|0| Removed|
|Combobox Converged.Disabled option.active option.chromium.png|0| Removed|
|Combobox Converged.Disabled option.hover.chromium.png|0| Removed|
|Combobox Converged.Disabled with value.focused.chromium.png|0| Removed|
|Combobox Converged.Disabled with value.hover.chromium.png|0| Removed|
|Combobox Converged.Disabled.focused.chromium.png|0| Removed|
|Combobox Converged.Disabled.hover.chromium.png|0| Removed|
|Combobox Converged.Invalid- filled-darker.focused.chromium.png|0| Removed|
|Combobox Converged.Invalid- filled-darker.hover.chromium.png|0| Removed|
|Combobox Converged.Invalid- filled-lighter.focused.chromium.png|0| Removed|
|Combobox Converged.Invalid- filled-lighter.hover.chromium.png|0| Removed|
|Combobox Converged.Invalid- outline.focused.chromium.png|0| Removed|
|Combobox Converged.Invalid- outline.hover.chromium.png|0| Removed|
|Combobox Converged.Invalid- underline.focused.chromium.png|0| Removed|
|Combobox Converged.Invalid- underline.hover.chromium.png|0| Removed|
|Combobox Converged.Open with grouped options.active option.chromium.png|0| Removed|
|Combobox Converged.Open with grouped options.hover.chromium.png|0| Removed|
|Combobox Converged.Open with inlinePopup.active option.chromium.png|0| Removed|
|Combobox Converged.Open with inlinePopup.hover.chromium.png|0| Removed|
|Combobox Converged.Open.active option.chromium.png|0| Removed|
|Combobox Converged.Open.hover.chromium.png|0| Removed|
|Combobox Converged.Option with long content.active option.chromium.png|0| Removed|
|Combobox Converged.Option with long content.hover.chromium.png|0| Removed|
|Combobox Converged.Size- large.focused.chromium.png|0| Removed|
|Combobox Converged.Size- large.hover.chromium.png|0| Removed|
|Combobox Converged.Size- small.focused.chromium.png|0| Removed|
|Combobox Converged.Size- small.hover.chromium.png|0| Removed|
|Combobox Converged.With placeholder.focused.chromium.png|0| Removed|
|Combobox Converged.With placeholder.hover.chromium.png|0| Removed|
|Combobox Converged.With selection.active option.chromium.png|0| Removed|
|Combobox Converged.With selection.hover.chromium.png|0| Removed|
|Combobox Converged.With value.focused.chromium.png|0| Removed|
|Combobox Converged.With value.hover.chromium.png|0| Removed|

</details>


  <details><summary><b>CompoundButton Converged</b> 60 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|CompoundButton Converged.Circular and icon only.default.chromium.png|1607199| Changed|
|CompoundButton Converged.Circular.default.chromium.png|1606824| Changed|
|CompoundButton Converged.Default - RTL.default.chromium.png|1606810| Changed|
|CompoundButton Converged.Default.default.chromium.png|1606813| Changed|
|CompoundButton Converged.Disabled.default.chromium.png|1606648| Changed|
|CompoundButton Converged.Outline.default.chromium.png|1606813| Changed|
|CompoundButton Converged.Icon only.default.chromium.png|1607193| Changed|
|CompoundButton Converged.Size large - with long text wrapping.chromium.png|1605494| Changed|
|CompoundButton Converged.Size large.default.chromium.png|1606626| Changed|
|CompoundButton Converged.Outline Disabled.default.chromium.png|1607210| Changed|
|CompoundButton Converged.Primary.default.chromium.png|1601248| Changed|
|CompoundButton Converged.Circular and icon only - RTL.default.chromium.png|1607198| Changed|
|CompoundButton Converged.Primary Disabled.default.chromium.png|1606677| Changed|
|CompoundButton Converged.Size medium - with long text wrapping.chromium.png|1605671| Changed|
|CompoundButton Converged.Size small.default.chromium.png|1606666| Changed|
|CompoundButton Converged.Transparent Disabled.default.chromium.png|1607266| Changed|
|CompoundButton Converged.Transparent.default.chromium.png|1606936| Changed|
|CompoundButton Converged.With icon after content - RTL.default.chromium.png|1606569| Changed|
|CompoundButton Converged.With icon before content.default.chromium.png|1606653| Changed|
|CompoundButton Converged.Subtle.default.chromium.png|1606936| Changed|
|CompoundButton Converged.Subtle Disabled.default.chromium.png|1607266| Changed|
|CompoundButton Converged.Size small - with long text wrapping.chromium.png|1606016| Changed|
|CompoundButton Converged.With icon after content.default.chromium.png|1606668| Changed|
|CompoundButton Converged.With icon before content - RTL.default.chromium.png|1606569| Changed|
|CompoundButton Converged.Circular and icon only - RTL.hover.chromium.png|0| Removed|
|CompoundButton Converged.Circular and icon only - RTL.pressed.chromium.png|0| Removed|
|CompoundButton Converged.Circular and icon only.hover.chromium.png|0| Removed|
|CompoundButton Converged.Circular and icon only.pressed.chromium.png|0| Removed|
|CompoundButton Converged.Circular.hover.chromium.png|0| Removed|
|CompoundButton Converged.Circular.pressed.chromium.png|0| Removed|
|CompoundButton Converged.Default - RTL.pressed.chromium.png|0| Removed|
|CompoundButton Converged.Default.pressed.chromium.png|0| Removed|
|CompoundButton Converged.Disabled.hover.chromium.png|0| Removed|
|CompoundButton Converged.Disabled.pressed.chromium.png|0| Removed|
|CompoundButton Converged.Icon only.hover.chromium.png|0| Removed|
|CompoundButton Converged.Icon only.pressed.chromium.png|0| Removed|
|CompoundButton Converged.Outline Disabled.hover.chromium.png|0| Removed|
|CompoundButton Converged.Outline Disabled.pressed.chromium.png|0| Removed|
|CompoundButton Converged.Outline.hover.chromium.png|0| Removed|
|CompoundButton Converged.Outline.pressed.chromium.png|0| Removed|
|CompoundButton Converged.Primary Disabled.hover.chromium.png|0| Removed|
|CompoundButton Converged.Primary Disabled.pressed.chromium.png|0| Removed|
|CompoundButton Converged.Primary.hover.chromium.png|0| Removed|
|CompoundButton Converged.Primary.pressed.chromium.png|0| Removed|
|CompoundButton Converged.Size large.hover.chromium.png|0| Removed|
|CompoundButton Converged.Size large.pressed.chromium.png|0| Removed|
|CompoundButton Converged.Size small.hover.chromium.png|0| Removed|
|CompoundButton Converged.Size small.pressed.chromium.png|0| Removed|
|CompoundButton Converged.Subtle Disabled.hover.chromium.png|0| Removed|
|CompoundButton Converged.Subtle Disabled.pressed.chromium.png|0| Removed|
|CompoundButton Converged.Subtle.hover.chromium.png|0| Removed|
|CompoundButton Converged.Subtle.pressed.chromium.png|0| Removed|
|CompoundButton Converged.Transparent Disabled.hover.chromium.png|0| Removed|
|CompoundButton Converged.Transparent Disabled.pressed.chromium.png|0| Removed|
|CompoundButton Converged.Transparent.hover.chromium.png|0| Removed|
|CompoundButton Converged.Transparent.pressed.chromium.png|0| Removed|
|CompoundButton Converged.With icon after content - RTL.pressed.chromium.png|0| Removed|
|CompoundButton Converged.With icon after content.pressed.chromium.png|0| Removed|
|CompoundButton Converged.With icon before content - RTL.pressed.chromium.png|0| Removed|
|CompoundButton Converged.With icon before content.pressed.chromium.png|0| Removed|

</details>


  <details><summary><b>DatePicker Compat</b> 23 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DatePicker Compat.DateRange- day.default.chromium.png|379230| Changed|
|DatePicker Compat.DateRange- month.default.chromium.png|376610| Changed|
|DatePicker Compat.DateRange- work week.default.chromium.png|378903| Changed|
|DatePicker Compat.DateRange- week.default.chromium.png|378726| Changed|
|DatePicker Compat.Required.default.chromium.png|392225| Changed|
|DatePicker Compat.DateRange- month - High Contrast.default.chromium.png|260471| Changed|
|DatePicker Compat.DateRange- month - Dark Mode.default.chromium.png|263318| Changed|
|DatePicker Compat.With label.default.chromium.png|392231| Changed|
|DatePicker Compat.Underlined.default.chromium.png|382339| Changed|
|DatePicker Compat.allowTextInput.default.chromium.png|382289| Changed|
|DatePicker Compat.Underlined and required.default.chromium.png|382339| Changed|
|DatePicker Compat.marked dates.default.chromium.png|379232| Changed|
|DatePicker Compat.default - High Contrast.rest.chromium.png|245299| Changed|
|DatePicker Compat.default - Dark Mode.rest.chromium.png|248391| Changed|
|DatePicker Compat.showWeekNumbers - Dark Mode.default.chromium.png|250228| Changed|
|DatePicker Compat.marked dates - High Contrast.default.chromium.png|249005| Changed|
|DatePicker Compat.marked dates - Dark Mode.default.chromium.png|250205| Changed|
|DatePicker Compat.showWeekNumbers.default.chromium.png|379089| Changed|
|DatePicker Compat.default.rest.chromium.png|382289| Changed|
|DatePicker Compat.showWeekNumbers - High Contrast.default.chromium.png|249321| Changed|
|DatePicker Compat.default - Dark Mode.opened.chromium.png|0| Removed|
|DatePicker Compat.default - High Contrast.opened.chromium.png|0| Removed|
|DatePicker Compat.default.opened.chromium.png|0| Removed|

</details>


  <details><summary><b>Dialog</b> 46 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dialog.Fluid Actions End.chromium.png|691260| Changed|
|Dialog.Fluid Actions Start.chromium.png|691260| Changed|
|Dialog.actions position start & position end - High Contrast.chromium.png|665819| Changed|
|Dialog.actions position start & position end.chromium.png|691260| Changed|
|Dialog.actions position start & position end - RTL.chromium.png|691260| Changed|
|Dialog.actions position start & position end - Dark Mode.chromium.png|669524| Changed|
|Dialog.actions position start - Dark Mode.chromium.png|669524| Changed|
|Dialog.actions position start - High Contrast.chromium.png|665819| Changed|
|Dialog.actions position start.chromium.png|691260| Changed|
|Dialog.alert.chromium.png|691260| Changed|
|Dialog.actions position start - RTL.chromium.png|691260| Changed|
|Dialog.alert - Dark Mode.chromium.png|669524| Changed|
|Dialog.default - High Contrast.chromium.png|665819| Changed|
|Dialog.alert - High Contrast.chromium.png|665819| Changed|
|Dialog.default.chromium.png|691260| Changed|
|Dialog.nested - Dark Mode.chromium.png|675860| Changed|
|Dialog.default - Dark Mode.chromium.png|669524| Changed|
|Dialog.nested - High Contrast.chromium.png|672195| Changed|
|Dialog.alert - RTL.chromium.png|691260| Changed|
|Dialog.default - RTL.chromium.png|691260| Changed|
|Dialog.nested - RTL.chromium.png|698111| Changed|
|Dialog.nested.chromium.png|698111| Changed|
|Dialog.no actions - High Contrast.chromium.png|675506| Changed|
|Dialog.no actions - RTL.chromium.png|700887| Changed|
|Dialog.no title & no actions - RTL.chromium.png|707626| Changed|
|Dialog.no actions - Dark Mode.chromium.png|679212| Changed|
|Dialog.no title & no actions - High Contrast.chromium.png|682289| Changed|
|Dialog.no actions.chromium.png|700887| Changed|
|Dialog.no title - Dark Mode.chromium.png|676306| Changed|
|Dialog.no title & no actions - Dark Mode.chromium.png|685994| Changed|
|Dialog.no title & no actions.chromium.png|707626| Changed|
|Dialog.no title - RTL.chromium.png|697999| Changed|
|Dialog.no title - High Contrast.chromium.png|672600| Changed|
|Dialog.no title.chromium.png|697999| Changed|
|Dialog.non-modal - RTL.chromium.png|24| Changed|
|Dialog.non-modal - Dark Mode.chromium.png|24| Changed|
|Dialog.non-modal - High Contrast.chromium.png|28| Changed|
|Dialog.non-modal.chromium.png|24| Changed|
|Dialog.scroll long content - High Contrast.chromium.png|487772| Changed|
|Dialog.scroll long content - RTL.chromium.png|505378| Changed|
|Dialog.title custom action - Dark Mode.chromium.png|678243| Changed|
|Dialog.title custom action - RTL.chromium.png|699924| Changed|
|Dialog.scroll long content.chromium.png|505378| Changed|
|Dialog.scroll long content - Dark Mode.chromium.png|490310| Changed|
|Dialog.title custom action - High Contrast.chromium.png|674538| Changed|
|Dialog.title custom action.chromium.png|699924| Changed|

</details>


  <details><summary><b>Divider Converged - Horizontal</b> 18 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Divider Converged - Horizontal.Appearance strong - Dark Mode.default.chromium.png|244507| Changed|
|Divider Converged - Horizontal.Appearance brand - Dark Mode.default.chromium.png|244523| Changed|
|Divider Converged - Horizontal.Appearance brand.default.chromium.png|247185| Changed|
|Divider Converged - Horizontal.Appearance strong - High Contrast.default.chromium.png|245321| Changed|
|Divider Converged - Horizontal.Appearance brand - High Contrast.default.chromium.png|245321| Changed|
|Divider Converged - Horizontal.Appearance strong.default.chromium.png|247292| Changed|
|Divider Converged - Horizontal.Appearance subtle - Dark Mode.default.chromium.png|244524| Changed|
|Divider Converged - Horizontal.Appearance subtle.default.chromium.png|247322| Changed|
|Divider Converged - Horizontal.with content - Dark Mode.default.chromium.png|244504| Changed|
|Divider Converged - Horizontal.Inset.default.chromium.png|247309| Changed|
|Divider Converged - Horizontal.End Aligned - RTL.default.chromium.png|247306| Changed|
|Divider Converged - Horizontal.Appearance subtle - High Contrast.default.chromium.png|245321| Changed|
|Divider Converged - Horizontal.Start Aligned.default.chromium.png|247306| Changed|
|Divider Converged - Horizontal.Start Aligned - RTL.default.chromium.png|247306| Changed|
|Divider Converged - Horizontal.with content - RTL.default.chromium.png|247306| Changed|
|Divider Converged - Horizontal.End Aligned.default.chromium.png|247306| Changed|
|Divider Converged - Horizontal.with content.default.chromium.png|247306| Changed|
|Divider Converged - Horizontal.with content - High Contrast.default.chromium.png|245321| Changed|

</details>


  <details><summary><b>Divider Converged - Vertical</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Divider Converged - Vertical.End Aligned.default.chromium.png|244507| Changed|
|Divider Converged - Vertical.Center Aligned.default.chromium.png|244507| Changed|
|Divider Converged - Vertical.inset.default.chromium.png|245286| Changed|
|Divider Converged - Vertical.Start Aligned.default.chromium.png|244507| Changed|

</details>


  <details><summary><b>Dropdown Converged</b> 34 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dropdown Converged.Disabled option.default.chromium.png|244766| Changed|
|Dropdown Converged.Appearance- outline (default).default.chromium.png|244790| Changed|
|Dropdown Converged.Appearance- underline.default.chromium.png|244621| Changed|
|Dropdown Converged.Invalid- underline.default.chromium.png|244623| Changed|
|Dropdown Converged.Appearance- filled-lighter.default.chromium.png|244106| Changed|
|Dropdown Converged.Invalid- filled-darker.default.chromium.png|243478| Changed|
|Dropdown Converged.Disabled.default.chromium.png|244936| Changed|
|Dropdown Converged.Invalid- filled-lighter.default.chromium.png|243767| Changed|
|Dropdown Converged.Invalid- outline.default.chromium.png|244662| Changed|
|Dropdown Converged.Appearance- filled-darker.default.chromium.png|243794| Changed|
|Dropdown Converged.Open with inlinePopup.default.chromium.png|244766| Changed|
|Dropdown Converged.Open with grouped options.default.chromium.png|244766| Changed|
|Dropdown Converged.Disabled with value.default.chromium.png|244917| Changed|
|Dropdown Converged.Open.default.chromium.png|244766| Changed|
|Dropdown Converged.Option with long content.default.chromium.png|244766| Changed|
|Dropdown Converged.Size- large.default.chromium.png|245505| Changed|
|Dropdown Converged.With multiselect selection.default.chromium.png|244766| Changed|
|Dropdown Converged.Size- small.default.chromium.png|243222| Changed|
|Dropdown Converged.With multiselect value.default.chromium.png|244608| Changed|
|Dropdown Converged.With placeholder.default.chromium.png|244649| Changed|
|Dropdown Converged.With selection.default.chromium.png|244766| Changed|
|Dropdown Converged.With value.default.chromium.png|244658| Changed|
|Dropdown Converged.Disabled option.active option.chromium.png|0| Removed|
|Dropdown Converged.Disabled option.hover.chromium.png|0| Removed|
|Dropdown Converged.Open with grouped options.active option.chromium.png|0| Removed|
|Dropdown Converged.Open with grouped options.hover.chromium.png|0| Removed|
|Dropdown Converged.Open with inlinePopup.active option.chromium.png|0| Removed|
|Dropdown Converged.Open with inlinePopup.hover.chromium.png|0| Removed|
|Dropdown Converged.Open.active option.chromium.png|0| Removed|
|Dropdown Converged.Open.hover.chromium.png|0| Removed|
|Dropdown Converged.Option with long content.active option.chromium.png|0| Removed|
|Dropdown Converged.Option with long content.hover.chromium.png|0| Removed|
|Dropdown Converged.With selection.active option.chromium.png|0| Removed|
|Dropdown Converged.With selection.hover.chromium.png|0| Removed|

</details>


  <details><summary><b>Field</b> 23 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Field.ProgressBar.default.chromium.png|1493| Changed|
|Field.Checkbox.default.chromium.png|273016| Changed|
|Field.RadioGroup.default.chromium.png|323450| Changed|
|Field.Dropdown.default.chromium.png|323815| Changed|
|Field.Combobox.default.chromium.png|323791| Changed|
|Field.Select.default.chromium.png|324085| Changed|
|Field.Switch.default.chromium.png|284082| Changed|
|Field.Slider.default.chromium.png|279221| Changed|
|Field.hint.default.chromium.png|263496| Changed|
|Field.Textarea.default.chromium.png|353397| Changed|
|Field.SpinButton.default.chromium.png|323204| Changed|
|Field.base.default.chromium.png|252534| Changed|
|Field.horizontal+noLabel.default.chromium.png|260676| Changed|
|Field.horizontal.default.chromium.png|247091| Changed|
|Field.horizontal+longLabel.default.chromium.png|273640| Changed|
|Field.size-large.default.chromium.png|256042| Changed|
|Field.required.default.chromium.png|252518| Changed|
|Field.size-small.default.chromium.png|249239| Changed|
|Field.validation-error.default.chromium.png|258935| Changed|
|Field.validation-none.default.chromium.png|259051| Changed|
|Field.validation-success.default.chromium.png|272776| Changed|
|Field.validationMessageIcon.default.chromium.png|258713| Changed|
|Field.validation-warning.default.chromium.png|259036| Changed|

</details>


  <details><summary><b>FluentProvider CustomStyleHooks</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|FluentProvider CustomStyleHooks.SplitButton.chromium.png|3357| Changed|
|FluentProvider CustomStyleHooks.CompoundButton.chromium.png|1602787| Changed|
|FluentProvider CustomStyleHooks.MenuButton.chromium.png|425| Changed|

</details>


  <details><summary><b>InfoButton</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|InfoButton.default - High Contrast.focus.chromium.png|240413| Changed|
|InfoButton.default.rest.chromium.png|259904| Changed|
|InfoButton.default.focus.chromium.png|259764| Changed|
|InfoButton.default - Dark Mode.focus.chromium.png|238307| Changed|
|InfoButton.default - High Contrast.rest.chromium.png|240342| Changed|
|InfoButton.default - Dark Mode.rest.chromium.png|238157| Changed|
|InfoButton.default - Dark Mode.active.chromium.png|0| Removed|
|InfoButton.default - Dark Mode.hover.chromium.png|0| Removed|
|InfoButton.default - High Contrast.active.chromium.png|0| Removed|
|InfoButton.default - High Contrast.hover.chromium.png|0| Removed|
|InfoButton.default.active.chromium.png|0| Removed|
|InfoButton.default.hover.chromium.png|0| Removed|

</details>


  <details><summary><b>InfoLabel</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|InfoLabel.default - High Contrast.rest.chromium.png|239467| Changed|
|InfoLabel.default - Dark Mode.rest.chromium.png|238695| Changed|
|InfoLabel.default.rest.chromium.png|242093| Changed|
|InfoLabel.required - RTL.rest.chromium.png|241978| Changed|
|InfoLabel.required.rest.chromium.png|241973| Changed|
|InfoLabel.size-small.rest.chromium.png|242134| Changed|
|InfoLabel.size-large.rest.chromium.png|242203| Changed|
|InfoLabel.default - RTL.rest.chromium.png|242094| Changed|
|InfoLabel.wrap.rest.chromium.png|248252| Changed|

</details>


  <details><summary><b>Input Converged</b> 47 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Input Converged.Appearance- filled-lighter.default.chromium.png|246996| Changed|
|Input Converged.Appearance- outline (default).default.chromium.png|244325| Changed|
|Input Converged.Appearance- underline.default.chromium.png|244371| Changed|
|Input Converged.Disabled, appearance- filled-darker.default.chromium.png|244524| Changed|
|Input Converged.Disabled, appearance- outline.default.chromium.png|244524| Changed|
|Input Converged.Disabled, appearance- underline.default.chromium.png|244555| Changed|
|Input Converged.Disabled, appearance- filled-lighter.default.chromium.png|246801| Changed|
|Input Converged.Appearance- filled-darker.default.chromium.png|244252| Changed|
|Input Converged.Invalid, appearance- underline.default.chromium.png|244372| Changed|
|Input Converged.Invalid, appearance- filled-darker.default.chromium.png|243994| Changed|
|Input Converged.Invalid, appearance- outline.default.chromium.png|244217| Changed|
|Input Converged.Size- large.default.chromium.png|244987| Changed|
|Input Converged.Inline.default.chromium.png|248780| Changed|
|Input Converged.Invalid, appearance- filled-lighter.default.chromium.png|246720| Changed|
|Input Converged.With appearance override.default.chromium.png|252190| Changed|
|Input Converged.contentBefore.default.chromium.png|244284| Changed|
|Input Converged.With value.default.chromium.png|244355| Changed|
|Input Converged.Size- small.default.chromium.png|242592| Changed|
|Input Converged.contentAfter - RTL.default.chromium.png|244289| Changed|
|Input Converged.contentAfter.default.chromium.png|244289| Changed|
|Input Converged.contentBefore - RTL.default.chromium.png|244284| Changed|
|Input Converged.Appearance- filled-darker.focused.chromium.png|0| Removed|
|Input Converged.Appearance- filled-darker.hover.chromium.png|0| Removed|
|Input Converged.Appearance- filled-lighter.focused.chromium.png|0| Removed|
|Input Converged.Appearance- filled-lighter.hover.chromium.png|0| Removed|
|Input Converged.Appearance- outline (default).focused.chromium.png|0| Removed|
|Input Converged.Appearance- outline (default).hover.chromium.png|0| Removed|
|Input Converged.Appearance- underline.focused.chromium.png|0| Removed|
|Input Converged.Appearance- underline.hover.chromium.png|0| Removed|
|Input Converged.Disabled, appearance- filled-darker.focused.chromium.png|0| Removed|
|Input Converged.Disabled, appearance- filled-darker.hover.chromium.png|0| Removed|
|Input Converged.Disabled, appearance- filled-lighter.focused.chromium.png|0| Removed|
|Input Converged.Disabled, appearance- filled-lighter.hover.chromium.png|0| Removed|
|Input Converged.Disabled, appearance- outline.focused.chromium.png|0| Removed|
|Input Converged.Disabled, appearance- outline.hover.chromium.png|0| Removed|
|Input Converged.Disabled, appearance- underline.focused.chromium.png|0| Removed|
|Input Converged.Disabled, appearance- underline.hover.chromium.png|0| Removed|
|Input Converged.Invalid, appearance- filled-darker.focused.chromium.png|0| Removed|
|Input Converged.Invalid, appearance- filled-darker.hover.chromium.png|0| Removed|
|Input Converged.Invalid, appearance- filled-lighter.focused.chromium.png|0| Removed|
|Input Converged.Invalid, appearance- filled-lighter.hover.chromium.png|0| Removed|
|Input Converged.Invalid, appearance- outline.focused.chromium.png|0| Removed|
|Input Converged.Invalid, appearance- outline.hover.chromium.png|0| Removed|
|Input Converged.Invalid, appearance- underline.focused.chromium.png|0| Removed|
|Input Converged.Invalid, appearance- underline.hover.chromium.png|0| Removed|
|Input Converged.With value.focused.chromium.png|0| Removed|
|Input Converged.With value.hover.chromium.png|0| Removed|

</details>


  <details><summary><b>Menu</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu.Nested Submenus Small Viewport Flipped.default.chromium.png|719| Changed|
|Menu.Nested Submenus Small Viewport Stacked.nested menu.chromium.png|790| Changed|
|Menu.Nested Submenus Small Viewport Flipped.nested menu.chromium.png|1227| Changed|
|Menu.Nested Submenus Small Viewport Stacked.default.chromium.png|719| Changed|

</details>


  <details><summary><b>Menu Converged - MenuItemLinks</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - MenuItemLinks.default.hover menuitemlink.chromium.png|203| Changed|
|Menu Converged - MenuItemLinks.default.chromium.png|198| Changed|

</details>


  <details><summary><b>Menu Converged - basic</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - basic.default.chromium.png|198| Changed|
|Menu Converged - basic.default - RTL.chromium.png|198| Changed|
|Menu Converged - basic.default - RTL.hover menuitem.chromium.png|203| Changed|
|Menu Converged - basic.default.hover menuitem.chromium.png|203| Changed|

</details>


  <details><summary><b>Menu Converged - groups</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - groups.default - High Contrast.chromium.png|533| Changed|
|Menu Converged - groups.default - Dark Mode.chromium.png|359| Changed|
|Menu Converged - groups.default.chromium.png|396| Changed|
|Menu Converged - groups.default - RTL.chromium.png|396| Changed|

</details>


  <details><summary><b>Menu Converged - icon slotted content</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - icon slotted content.default - RTL.chromium.png|539| Changed|
|Menu Converged - icon slotted content.default - RTL.hover menuitem.chromium.png|544| Changed|
|Menu Converged - icon slotted content.default.chromium.png|429| Changed|
|Menu Converged - icon slotted content.default.hover menuitem.chromium.png|430| Changed|

</details>


  <details><summary><b>Menu Converged - nested submenus</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - nested submenus.default - RTL.all open.chromium.png|1072| Changed|
|Menu Converged - nested submenus.default - RTL.chromium.png|613| Changed|
|Menu Converged - nested submenus.default.all open.chromium.png|1072| Changed|
|Menu Converged - nested submenus.default.chromium.png|613| Changed|

</details>


  <details><summary><b>Menu Converged - secondary content</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - secondary content.default - RTL.chromium.png|414| Changed|
|Menu Converged - secondary content.default.chromium.png|362| Changed|
|Menu Converged - secondary content.default - RTL.hover menuitem.chromium.png|419| Changed|
|Menu Converged - secondary content.default.hover menuitem.chromium.png|363| Changed|

</details>


  <details><summary><b>Menu Converged - selection</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - selection.checkbox - Dark Mode.chromium.png|407| Changed|
|Menu Converged - selection.checkbox - High Contrast.selected.chromium.png|552| Changed|
|Menu Converged - selection.checkbox - Dark Mode.selected.chromium.png|384| Changed|
|Menu Converged - selection.checkbox - RTL.selected.chromium.png|453| Changed|
|Menu Converged - selection.checkbox - High Contrast.chromium.png|603| Changed|
|Menu Converged - selection.checkbox.chromium.png|448| Changed|
|Menu Converged - selection.checkbox - RTL.chromium.png|448| Changed|
|Menu Converged - selection.checkbox.selected.chromium.png|445| Changed|

</details>


  <details><summary><b>Menu Converged - selection groups</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - selection groups.default - RTL.selected.chromium.png|947| Changed|
|Menu Converged - selection groups.default.chromium.png|942| Changed|
|Menu Converged - selection groups.default - RTL.chromium.png|942| Changed|
|Menu Converged - selection groups.default.selected.chromium.png|939| Changed|

</details>


  <details><summary><b>Menu Converged - split item</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - split item.default.chromium.png|533| Changed|
|Menu Converged - split item.default - RTL.submenu open.chromium.png|1020| Changed|
|Menu Converged - split item.default - RTL.chromium.png|534| Changed|
|Menu Converged - split item.default.submenu open.chromium.png|1019| Changed|

</details>


  <details><summary><b>Menu Converged - submenuIndicator slotted content</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - submenuIndicator slotted content.default - RTL.chromium.png|141| Changed|
|Menu Converged - submenuIndicator slotted content.default - RTL.submenus open.chromium.png|840| Changed|
|Menu Converged - submenuIndicator slotted content.default.chromium.png|146| Changed|
|Menu Converged - submenuIndicator slotted content.default.submenus open.chromium.png|840| Changed|

</details>


  <details><summary><b>Menu nested within a MenuTrigger</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu nested within a MenuTrigger.default.chromium.png|490| Changed|
|Menu nested within a MenuTrigger.default.submenu open.chromium.png|997| Changed|

</details>


  <details><summary><b>MenuButton Converged</b> 48 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|MenuButton Converged.Circular.hover.chromium.png|27| Changed|
|MenuButton Converged.Circular.default.chromium.png|24| Changed|
|MenuButton Converged.Circular.pressed.chromium.png|83| Changed|
|MenuButton Converged.Default - RTL.hover.chromium.png|24| Changed|
|MenuButton Converged.Default.hover.chromium.png|24| Changed|
|MenuButton Converged.Default - RTL.pressed.chromium.png|82| Changed|
|MenuButton Converged.Default - RTL.default.chromium.png|22| Changed|
|MenuButton Converged.Default.pressed.chromium.png|82| Changed|
|MenuButton Converged.Disabled.default.chromium.png|36| Changed|
|MenuButton Converged.Outline Disabled.hover.chromium.png|12| Changed|
|MenuButton Converged.Default.default.chromium.png|22| Changed|
|MenuButton Converged.Disabled.hover.chromium.png|36| Changed|
|MenuButton Converged.Outline.hover.chromium.png|25| Changed|
|MenuButton Converged.Outline Disabled.pressed.chromium.png|12| Changed|
|MenuButton Converged.Outline Disabled.default.chromium.png|12| Changed|
|MenuButton Converged.Disabled.pressed.chromium.png|36| Changed|
|MenuButton Converged.Primary Disabled.default.chromium.png|31| Changed|
|MenuButton Converged.Primary Disabled.hover.chromium.png|31| Changed|
|MenuButton Converged.Size large.default.chromium.png|30| Changed|
|MenuButton Converged.Outline.pressed.chromium.png|33| Changed|
|MenuButton Converged.Outline.default.chromium.png|22| Changed|
|MenuButton Converged.Primary Disabled.pressed.chromium.png|31| Changed|
|MenuButton Converged.Primary.hover.chromium.png|356| Changed|
|MenuButton Converged.Primary.default.chromium.png|335| Changed|
|MenuButton Converged.Primary.pressed.chromium.png|417| Changed|
|MenuButton Converged.Size large - with long text wrapping.chromium.png|1978| Changed|
|MenuButton Converged.Size large.pressed.chromium.png|125| Changed|
|MenuButton Converged.Size medium - with long text wrapping.chromium.png|1228| Changed|
|MenuButton Converged.Size large.hover.chromium.png|33| Changed|
|MenuButton Converged.Size small.default.chromium.png|19| Changed|
|MenuButton Converged.Size small - with long text wrapping.chromium.png|912| Changed|
|MenuButton Converged.Size small.hover.chromium.png|21| Changed|
|MenuButton Converged.Size small.pressed.chromium.png|64| Changed|
|MenuButton Converged.Subtle Disabled.hover.chromium.png|2| Changed|
|MenuButton Converged.Subtle Disabled.default.chromium.png|2| Changed|
|MenuButton Converged.Subtle.hover.chromium.png|6| Changed|
|MenuButton Converged.Subtle Disabled.pressed.chromium.png|2| Changed|
|MenuButton Converged.Transparent Disabled.default.chromium.png|2| Changed|
|MenuButton Converged.Subtle.default.chromium.png|5| Changed|
|MenuButton Converged.Subtle.pressed.chromium.png|67| Changed|
|MenuButton Converged.Transparent Disabled.hover.chromium.png|2| Changed|
|MenuButton Converged.Transparent Disabled.pressed.chromium.png|2| Changed|
|MenuButton Converged.Transparent.pressed.chromium.png|5| Changed|
|MenuButton Converged.Transparent.hover.chromium.png|4| Changed|
|MenuButton Converged.Transparent.default.chromium.png|5| Changed|
|MenuButton Converged.With icon.default.chromium.png|22| Changed|
|MenuButton Converged.With icon.hover.chromium.png|24| Changed|
|MenuButton Converged.With icon.pressed.chromium.png|82| Changed|

</details>


  <details><summary><b>Persona Converged</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Persona Converged.basic - Dark Mode.normal.chromium.png|8489| Changed|
|Persona Converged.basic - High Contrast.normal.chromium.png|1966| Changed|
|Persona Converged.basic.normal.chromium.png|45464| Changed|
|Persona Converged.basic - RTL.normal.chromium.png|45459| Changed|
|Persona Converged.textPosition.normal.chromium.png|25031| Changed|
|Persona Converged.textAlignment.normal.chromium.png|44149| Changed|
|Persona Converged.textWrap.normal.chromium.png|66389| Changed|
|Persona Converged.size-presence.normal.chromium.png|96704| Changed|
|Persona Converged.size-avatar.normal.chromium.png|232991| Changed|

</details>


  <details><summary><b>PresenceBadge Converged - OOF status</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|PresenceBadge Converged - OOF status.default.chromium.png|395| Changed|
|PresenceBadge Converged - OOF status.default - RTL.chromium.png|395| Changed|

</details>


  <details><summary><b>PresenceBadge Converged - inverted background</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|PresenceBadge Converged - inverted background.default - High Contrast.default.chromium.png|18925| Changed|
|PresenceBadge Converged - inverted background.default - Dark Mode.default.chromium.png|29466| Changed|
|PresenceBadge Converged - inverted background.default.default.chromium.png|20534| Changed|

</details>


  <details><summary><b>PresenceBadge Converged - sizes</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|PresenceBadge Converged - sizes.default - RTL.chromium.png|740| Changed|
|PresenceBadge Converged - sizes.default.chromium.png|740| Changed|

</details>


  <details><summary><b>PresenceBadge Converged - status</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|PresenceBadge Converged - status.default - RTL.chromium.png|584| Changed|
|PresenceBadge Converged - status.default.chromium.png|583| Changed|

</details>


  <details><summary><b>ProgressBar converged</b> 11 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ProgressBar converged.Determinate + thickness - High Contrast.default.chromium.png|610| Changed|
|ProgressBar converged.Determinate + thickness - RTL.default.chromium.png|491| Changed|
|ProgressBar converged.Error.default.chromium.png|152| Changed|
|ProgressBar converged.Determinate + thickness.default.chromium.png|491| Changed|
|ProgressBar converged.Determinate + thickness - Dark Mode.default.chromium.png|318| Changed|
|ProgressBar converged.Warning.default.chromium.png|152| Changed|
|ProgressBar converged.Success.default.chromium.png|352| Changed|
|ProgressBar converged.Indeterminate + thickness.chromium.png|163| Changed|
|ProgressBar converged.Indeterminate + thickness - RTL.chromium.png|163| Changed|
|ProgressBar converged.Indeterminate + thickness - High Contrast.chromium.png|202| Changed|
|ProgressBar converged.Indeterminate + thickness - Dark Mode.chromium.png|104| Changed|

</details>


  <details><summary><b>Radio Converged</b> 48 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Radio Converged.disabled - Dark Mode.rest.chromium.png|238002| Changed|
|Radio Converged.checked.rest.chromium.png|244691| Changed|
|Radio Converged.disabled - High Contrast.rest.chromium.png|239362| Changed|
|Radio Converged.checked - Dark Mode.rest.chromium.png|238149| Changed|
|Radio Converged.checked - High Contrast.rest.chromium.png|239441| Changed|
|Radio Converged.disabled+checked - Dark Mode.default.chromium.png|238058| Changed|
|Radio Converged.disabled+checked.default.chromium.png|244800| Changed|
|Radio Converged.disabled+checked - High Contrast.default.chromium.png|239425| Changed|
|Radio Converged.disabled.rest.chromium.png|244862| Changed|
|Radio Converged.group-horizontal-stacked.default.chromium.png|250239| Changed|
|Radio Converged.group-vertical.default.chromium.png|258729| Changed|
|Radio Converged.group-disabled - RTL.default.chromium.png|259005| Changed|
|Radio Converged.group-horizontal-stacked - RTL.default.chromium.png|250275| Changed|
|Radio Converged.group-disabled.default.chromium.png|259005| Changed|
|Radio Converged.label-below - RTL.default.chromium.png|250485| Changed|
|Radio Converged.group-vertical - RTL.default.chromium.png|258734| Changed|
|Radio Converged.group-horizontal.default.chromium.png|244443| Changed|
|Radio Converged.group-horizontal - RTL.default.chromium.png|244443| Changed|
|Radio Converged.label-after.default.chromium.png|244763| Changed|
|Radio Converged.label-below.default.chromium.png|250473| Changed|
|Radio Converged.label-after - RTL.default.chromium.png|244763| Changed|
|Radio Converged.label-wrapping - RTL.default.chromium.png|256580| Changed|
|Radio Converged.label-wrapping.default.chromium.png|256581| Changed|
|Radio Converged.no-label - RTL.default.chromium.png|244891| Changed|
|Radio Converged.no-label.default.chromium.png|244891| Changed|
|Radio Converged.required - RTL.default.chromium.png|244773| Changed|
|Radio Converged.unchecked - High Contrast.rest.chromium.png|239446| Changed|
|Radio Converged.required.default.chromium.png|244773| Changed|
|Radio Converged.unchecked - Dark Mode.rest.chromium.png|238090| Changed|
|Radio Converged.unchecked.rest.chromium.png|244756| Changed|
|Radio Converged.checked - Dark Mode.active.chromium.png|0| Removed|
|Radio Converged.checked - Dark Mode.hover.chromium.png|0| Removed|
|Radio Converged.checked - High Contrast.active.chromium.png|0| Removed|
|Radio Converged.checked - High Contrast.hover.chromium.png|0| Removed|
|Radio Converged.checked.active.chromium.png|0| Removed|
|Radio Converged.checked.hover.chromium.png|0| Removed|
|Radio Converged.disabled - Dark Mode.active.chromium.png|0| Removed|
|Radio Converged.disabled - Dark Mode.hover.chromium.png|0| Removed|
|Radio Converged.disabled - High Contrast.active.chromium.png|0| Removed|
|Radio Converged.disabled - High Contrast.hover.chromium.png|0| Removed|
|Radio Converged.disabled.active.chromium.png|0| Removed|
|Radio Converged.disabled.hover.chromium.png|0| Removed|
|Radio Converged.unchecked - Dark Mode.active.chromium.png|0| Removed|
|Radio Converged.unchecked - Dark Mode.hover.chromium.png|0| Removed|
|Radio Converged.unchecked - High Contrast.active.chromium.png|0| Removed|
|Radio Converged.unchecked - High Contrast.hover.chromium.png|0| Removed|
|Radio Converged.unchecked.active.chromium.png|0| Removed|
|Radio Converged.unchecked.hover.chromium.png|0| Removed|

</details>


  <details><summary><b>SearchBox Converged</b> 49 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SearchBox Converged.Clears value.default.chromium.png|242304| Changed|
|SearchBox Converged.Appearance- filled-darker.default.chromium.png|242189| Changed|
|SearchBox Converged.Appearance- underline.default.chromium.png|242267| Changed|
|SearchBox Converged.Appearance- outline (default).default.chromium.png|242274| Changed|
|SearchBox Converged.Appearance- filled-lighter.default.chromium.png|245297| Changed|
|SearchBox Converged.Clears value.input focused.chromium.png|242429| Changed|
|SearchBox Converged.Disabled, appearance- filled-darker.default.chromium.png|242503| Changed|
|SearchBox Converged.Disabled, appearance- outline.default.chromium.png|242503| Changed|
|SearchBox Converged.WIth contentAfter - RTL.default.chromium.png|242274| Changed|
|SearchBox Converged.WIth contentAfter.default.chromium.png|242274| Changed|
|SearchBox Converged.Size- small.default.chromium.png|240332| Changed|
|SearchBox Converged.Size- large.default.chromium.png|243843| Changed|
|SearchBox Converged.Disabled, appearance- underline.default.chromium.png|242479| Changed|
|SearchBox Converged.Disabled, appearance- filled-lighter.default.chromium.png|245074| Changed|
|SearchBox Converged.With value.default.chromium.png|242304| Changed|
|SearchBox Converged.With custom dismiss.default.chromium.png|242274| Changed|
|SearchBox Converged.With custom dismiss - RTL.default.chromium.png|242274| Changed|
|SearchBox Converged.Appearance- filled-darker.focused.chromium.png|0| Removed|
|SearchBox Converged.Appearance- filled-darker.hover.chromium.png|0| Removed|
|SearchBox Converged.Appearance- filled-lighter.focused.chromium.png|0| Removed|
|SearchBox Converged.Appearance- filled-lighter.hover.chromium.png|0| Removed|
|SearchBox Converged.Appearance- outline (default).focused.chromium.png|0| Removed|
|SearchBox Converged.Appearance- outline (default).hover.chromium.png|0| Removed|
|SearchBox Converged.Appearance- underline.focused.chromium.png|0| Removed|
|SearchBox Converged.Appearance- underline.hover.chromium.png|0| Removed|
|SearchBox Converged.Clears value.dismiss clicked.chromium.png|0| Removed|
|SearchBox Converged.Clears value.dismiss focused.chromium.png|0| Removed|
|SearchBox Converged.Disabled, appearance- filled-darker.focused.chromium.png|0| Removed|
|SearchBox Converged.Disabled, appearance- filled-darker.hover.chromium.png|0| Removed|
|SearchBox Converged.Disabled, appearance- filled-lighter.focused.chromium.png|0| Removed|
|SearchBox Converged.Disabled, appearance- filled-lighter.hover.chromium.png|0| Removed|
|SearchBox Converged.Disabled, appearance- outline.focused.chromium.png|0| Removed|
|SearchBox Converged.Disabled, appearance- outline.hover.chromium.png|0| Removed|
|SearchBox Converged.Disabled, appearance- underline.focused.chromium.png|0| Removed|
|SearchBox Converged.Disabled, appearance- underline.hover.chromium.png|0| Removed|
|SearchBox Converged.Size- large.focused.chromium.png|0| Removed|
|SearchBox Converged.Size- large.hover.chromium.png|0| Removed|
|SearchBox Converged.Size- small.focused.chromium.png|0| Removed|
|SearchBox Converged.Size- small.hover.chromium.png|0| Removed|
|SearchBox Converged.WIth contentAfter - RTL.focused.chromium.png|0| Removed|
|SearchBox Converged.WIth contentAfter - RTL.hover.chromium.png|0| Removed|
|SearchBox Converged.WIth contentAfter.focused.chromium.png|0| Removed|
|SearchBox Converged.WIth contentAfter.hover.chromium.png|0| Removed|
|SearchBox Converged.With custom dismiss - RTL.focused.chromium.png|0| Removed|
|SearchBox Converged.With custom dismiss - RTL.hover.chromium.png|0| Removed|
|SearchBox Converged.With custom dismiss.focused.chromium.png|0| Removed|
|SearchBox Converged.With custom dismiss.hover.chromium.png|0| Removed|
|SearchBox Converged.With value.focused.chromium.png|0| Removed|
|SearchBox Converged.With value.hover.chromium.png|0| Removed|

</details>


  <details><summary><b>Select Converged</b> 30 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Select Converged.Custom Icon.default.chromium.png|244479| Changed|
|Select Converged.Disabled.default.chromium.png|244733| Changed|
|Select Converged.Appearance- filled-lighter.default.chromium.png|244648| Changed|
|Select Converged.Appearance- underline.default.chromium.png|244549| Changed|
|Select Converged.Appearance- filled-darker.default.chromium.png|244302| Changed|
|Select Converged.Appearance- outline (default).default.chromium.png|244486| Changed|
|Select Converged.Invalid- filled-lighter.default.chromium.png|244273| Changed|
|Select Converged.Invalid- filled-darker.default.chromium.png|243951| Changed|
|Select Converged.Invalid- outline.default.chromium.png|244338| Changed|
|Select Converged.Size- large.default.chromium.png|245192| Changed|
|Select Converged.Size- small.default.chromium.png|242711| Changed|
|Select Converged.Invalid- underline.default.chromium.png|244552| Changed|
|Select Converged.With value.default.chromium.png|244448| Changed|
|Select Converged.With appearance override.default.chromium.png|252064| Changed|
|Select Converged.Appearance- filled-darker.focused.chromium.png|0| Removed|
|Select Converged.Appearance- filled-darker.hover.chromium.png|0| Removed|
|Select Converged.Appearance- filled-lighter.focused.chromium.png|0| Removed|
|Select Converged.Appearance- filled-lighter.hover.chromium.png|0| Removed|
|Select Converged.Appearance- outline (default).focused.chromium.png|0| Removed|
|Select Converged.Appearance- outline (default).hover.chromium.png|0| Removed|
|Select Converged.Appearance- underline.focused.chromium.png|0| Removed|
|Select Converged.Appearance- underline.hover.chromium.png|0| Removed|
|Select Converged.Invalid- filled-darker.focused.chromium.png|0| Removed|
|Select Converged.Invalid- filled-darker.hover.chromium.png|0| Removed|
|Select Converged.Invalid- filled-lighter.focused.chromium.png|0| Removed|
|Select Converged.Invalid- filled-lighter.hover.chromium.png|0| Removed|
|Select Converged.Invalid- outline.focused.chromium.png|0| Removed|
|Select Converged.Invalid- outline.hover.chromium.png|0| Removed|
|Select Converged.Invalid- underline.focused.chromium.png|0| Removed|
|Select Converged.Invalid- underline.hover.chromium.png|0| Removed|

</details>


  <details><summary><b>Shadow DOM</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Shadow DOM.Default.default.chromium.png|1598531| Changed|
|Shadow DOM.Portal.normal.chromium.png|479| Changed|

</details>


  <details><summary><b>Slider Converged</b> 38 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Slider Converged.Root - High Contrast.default.chromium.png|237673| Changed|
|Slider Converged.Disabled - High Contrast.default.chromium.png|237461| Changed|
|Slider Converged.Disabled - Dark Mode.default.chromium.png|236732| Changed|
|Slider Converged.Horizontal - 0%.default.chromium.png|239279| Changed|
|Slider Converged.Vertical - 0% - RTL.default.chromium.png|238180| Changed|
|Slider Converged.Disabled Vertical.default.chromium.png|238426| Changed|
|Slider Converged.Disabled.default.chromium.png|239524| Changed|
|Slider Converged.Horizontal - 100%.default.chromium.png|239267| Changed|
|Slider Converged.Horizontal - 100% - RTL.default.chromium.png|239267| Changed|
|Slider Converged.Root - Dark Mode.default.chromium.png|236901| Changed|
|Slider Converged.Root - RTL.default.chromium.png|239301| Changed|
|Slider Converged.Vertical - 100% - RTL.default.chromium.png|238170| Changed|
|Slider Converged.Root.default.chromium.png|239301| Changed|
|Slider Converged.Horizontal - 0% - RTL.default.chromium.png|239278| Changed|
|Slider Converged.Vertical - 0%.default.chromium.png|238180| Changed|
|Slider Converged.Vertical - RTL.default.chromium.png|238204| Changed|
|Slider Converged.Vertical - 100%.default.chromium.png|238170| Changed|
|Slider Converged.Vertical.default.chromium.png|238204| Changed|
|Slider Converged.Disabled - Dark Mode.hover.chromium.png|0| Removed|
|Slider Converged.Disabled - Dark Mode.pressed.chromium.png|0| Removed|
|Slider Converged.Disabled - High Contrast.hover.chromium.png|0| Removed|
|Slider Converged.Disabled - High Contrast.pressed.chromium.png|0| Removed|
|Slider Converged.Disabled Vertical.hover.chromium.png|0| Removed|
|Slider Converged.Disabled Vertical.pressed.chromium.png|0| Removed|
|Slider Converged.Disabled.hover.chromium.png|0| Removed|
|Slider Converged.Disabled.pressed.chromium.png|0| Removed|
|Slider Converged.Root - Dark Mode.hover.chromium.png|0| Removed|
|Slider Converged.Root - Dark Mode.pressed.chromium.png|0| Removed|
|Slider Converged.Root - High Contrast.hover.chromium.png|0| Removed|
|Slider Converged.Root - High Contrast.pressed.chromium.png|0| Removed|
|Slider Converged.Root - RTL.hover.chromium.png|0| Removed|
|Slider Converged.Root - RTL.pressed.chromium.png|0| Removed|
|Slider Converged.Root.hover.chromium.png|0| Removed|
|Slider Converged.Root.pressed.chromium.png|0| Removed|
|Slider Converged.Vertical - RTL.hover.chromium.png|0| Removed|
|Slider Converged.Vertical - RTL.pressed.chromium.png|0| Removed|
|Slider Converged.Vertical.hover.chromium.png|0| Removed|
|Slider Converged.Vertical.pressed.chromium.png|0| Removed|

</details>


  <details><summary><b>SpinButton Converged</b> 263 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.rest.chromium.png|238214| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.rest.chromium.png|241400| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.rest.chromium.png|242626| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.rest.chromium.png|239361| Changed|
|SpinButton Converged.Appearance- filled-lighter - RTL.rest.chromium.png|247164| Changed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.rest.chromium.png|237868| Changed|
|SpinButton Converged.Appearance- filled-darker - RTL.rest.chromium.png|244383| Changed|
|SpinButton Converged.Appearance- filled-lighter.rest.chromium.png|247164| Changed|
|SpinButton Converged.Appearance- filled-darker.rest.chromium.png|244383| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.rest.chromium.png|239361| Changed|
|SpinButton Converged.Appearance- outline (default) - RTL.rest.chromium.png|244468| Changed|
|SpinButton Converged.Appearance- outline (default).rest.chromium.png|244468| Changed|
|SpinButton Converged.Appearance- underline - Dark Mode.rest.chromium.png|237901| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.rest.chromium.png|239285| Changed|
|SpinButton Converged.At Max Bound - High Contrast.rest.chromium.png|239360| Changed|
|SpinButton Converged.Appearance- underline - RTL.rest.chromium.png|244515| Changed|
|SpinButton Converged.At Max Bound - Dark Mode.rest.chromium.png|237866| Changed|
|SpinButton Converged.At Max Bound - RTL.rest.chromium.png|244471| Changed|
|SpinButton Converged.At Max Bound.rest.chromium.png|244471| Changed|
|SpinButton Converged.Appearance- underline.rest.chromium.png|244515| Changed|
|SpinButton Converged.At Min Bound - Dark Mode.rest.chromium.png|237864| Changed|
|SpinButton Converged.At Min Bound - High Contrast.rest.chromium.png|239358| Changed|
|SpinButton Converged.At Min Bound - RTL.rest.chromium.png|244471| Changed|
|SpinButton Converged.Custom Width - RTL.chromium.png|1607829| Changed|
|SpinButton Converged.At Min Bound.rest.chromium.png|244471| Changed|
|SpinButton Converged.Custom Width - High Contrast.chromium.png|1601332| Changed|
|SpinButton Converged.Disabled - Dark Mode.rest.chromium.png|237756| Changed|
|SpinButton Converged.Disabled - RTL.rest.chromium.png|244625| Changed|
|SpinButton Converged.Disabled - High Contrast.rest.chromium.png|239181| Changed|
|SpinButton Converged.Custom Width - Dark Mode.chromium.png|1600004| Changed|
|SpinButton Converged.Display Value - High Contrast.rest.chromium.png|239409| Changed|
|SpinButton Converged.Custom Width.chromium.png|1607829| Changed|
|SpinButton Converged.Display Value - RTL.rest.chromium.png|244388| Changed|
|SpinButton Converged.Disabled.rest.chromium.png|244625| Changed|
|SpinButton Converged.Invalid- outline.rest.chromium.png|244356| Changed|
|SpinButton Converged.Display Value - Dark Mode.rest.chromium.png|237950| Changed|
|SpinButton Converged.Invalid- filled-lighter.rest.chromium.png|246873| Changed|
|SpinButton Converged.Invalid- underline.rest.chromium.png|244517| Changed|
|SpinButton Converged.Size- medium (default).default.chromium.png|244468| Changed|
|SpinButton Converged.Size- small - Dark Mode.default.chromium.png|238076| Changed|
|SpinButton Converged.Size- medium (default) - RTL.default.chromium.png|244468| Changed|
|SpinButton Converged.Invalid- filled-darker.rest.chromium.png|244111| Changed|
|SpinButton Converged.Display Value.rest.chromium.png|244388| Changed|
|SpinButton Converged.Size- small - RTL.default.chromium.png|242706| Changed|
|SpinButton Converged.With appearance override.default.chromium.png|252668| Changed|
|SpinButton Converged.Size- small - High Contrast.default.chromium.png|239229| Changed|
|SpinButton Converged.Size- small.default.chromium.png|242707| Changed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.focused.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.focused.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker - RTL.focused.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker - RTL.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker - RTL.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker - RTL.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker - RTL.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker - RTL.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker.focused.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.focused.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.focused.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter - RTL.focused.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter - RTL.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter - RTL.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter - RTL.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter - RTL.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter - RTL.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter.focused.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.focused.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.focused.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default) - RTL.focused.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default) - RTL.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default) - RTL.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default) - RTL.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default) - RTL.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default) - RTL.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default).focused.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default).hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default).hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default).hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default).mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default).mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline - Dark Mode.focused.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline - Dark Mode.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline - Dark Mode.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline - Dark Mode.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline - Dark Mode.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline - Dark Mode.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline - High Contrast.focused.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline - High Contrast.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline - High Contrast.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline - High Contrast.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline - High Contrast.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline - High Contrast.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline - RTL.focused.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline - RTL.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline - RTL.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline - RTL.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline - RTL.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline - RTL.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline.focused.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound - Dark Mode.focused.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound - Dark Mode.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound - Dark Mode.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound - Dark Mode.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound - Dark Mode.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound - Dark Mode.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound - High Contrast.focused.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound - High Contrast.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound - High Contrast.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound - High Contrast.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound - High Contrast.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound - High Contrast.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound - RTL.focused.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound - RTL.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound - RTL.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound - RTL.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound - RTL.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound - RTL.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound.focused.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - Dark Mode.focused.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - Dark Mode.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - Dark Mode.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - Dark Mode.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - Dark Mode.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - Dark Mode.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - High Contrast.focused.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - High Contrast.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - High Contrast.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - High Contrast.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - High Contrast.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - High Contrast.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - RTL.focused.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - RTL.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - RTL.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - RTL.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - RTL.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - RTL.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound.focused.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Disabled - Dark Mode.focused.chromium.png|0| Removed|
|SpinButton Converged.Disabled - Dark Mode.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Disabled - Dark Mode.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Disabled - Dark Mode.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Disabled - Dark Mode.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Disabled - Dark Mode.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Disabled - High Contrast.focused.chromium.png|0| Removed|
|SpinButton Converged.Disabled - High Contrast.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Disabled - High Contrast.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Disabled - High Contrast.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Disabled - High Contrast.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Disabled - High Contrast.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Disabled - RTL.focused.chromium.png|0| Removed|
|SpinButton Converged.Disabled - RTL.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Disabled - RTL.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Disabled - RTL.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Disabled - RTL.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Disabled - RTL.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Disabled.focused.chromium.png|0| Removed|
|SpinButton Converged.Disabled.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Disabled.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Disabled.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Disabled.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Disabled.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Display Value - Dark Mode.focused.chromium.png|0| Removed|
|SpinButton Converged.Display Value - Dark Mode.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Display Value - Dark Mode.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Display Value - Dark Mode.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Display Value - Dark Mode.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Display Value - Dark Mode.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Display Value - High Contrast.focused.chromium.png|0| Removed|
|SpinButton Converged.Display Value - High Contrast.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Display Value - High Contrast.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Display Value - High Contrast.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Display Value - High Contrast.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Display Value - High Contrast.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Display Value - RTL.focused.chromium.png|0| Removed|
|SpinButton Converged.Display Value - RTL.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Display Value - RTL.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Display Value - RTL.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Display Value - RTL.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Display Value - RTL.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Display Value.focused.chromium.png|0| Removed|
|SpinButton Converged.Display Value.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Display Value.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Display Value.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Display Value.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Display Value.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- filled-darker.focused.chromium.png|0| Removed|
|SpinButton Converged.Invalid- filled-darker.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- filled-darker.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- filled-darker.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Invalid- filled-darker.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- filled-darker.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- filled-lighter.focused.chromium.png|0| Removed|
|SpinButton Converged.Invalid- filled-lighter.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- filled-lighter.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- filled-lighter.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Invalid- filled-lighter.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- filled-lighter.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- outline.focused.chromium.png|0| Removed|
|SpinButton Converged.Invalid- outline.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- outline.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- outline.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Invalid- outline.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- outline.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- underline.focused.chromium.png|0| Removed|
|SpinButton Converged.Invalid- underline.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- underline.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- underline.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Invalid- underline.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- underline.mouseDownIncrement.chromium.png|0| Removed|

</details>


  <details><summary><b>Spinner converged</b> 14 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Spinner converged.Inverted - High Contrast.chromium.png|9487| Changed|
|Spinner converged.Inverted with Label - High Contrast.chromium.png|31406| Changed|
|Spinner converged.Primary + size.chromium.png|3967| Changed|
|Spinner converged.Inverted with Label.chromium.png|20383| Changed|
|Spinner converged.Inverted with Label - Dark Mode.chromium.png|20383| Changed|
|Spinner converged.Inverted - Dark Mode.chromium.png|6158| Changed|
|Spinner converged.Primary with Label - Dark Mode.chromium.png|26174| Changed|
|Spinner converged.Inverted.chromium.png|6158| Changed|
|Spinner converged.Primary - Dark Mode.chromium.png|7916| Changed|
|Spinner converged.Primary.chromium.png|182| Changed|
|Spinner converged.Primary with Label.chromium.png|2149| Changed|
|Spinner converged.Primary with Label - High Contrast.chromium.png|31289| Changed|
|Spinner converged.Primary with Label - RTL.chromium.png|2113| Changed|
|Spinner converged.Primary - High Contrast.chromium.png|9263| Changed|

</details>


  <details><summary><b>Switch Converged</b> 66 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Switch Converged.Disabled and checked - Dark Mode.default.chromium.png|237078| Changed|
|Switch Converged.Disabled and checked - High Contrast.default.chromium.png|237846| Changed|
|Switch Converged.Disabled and unchecked - Dark Mode.default.chromium.png|237053| Changed|
|Switch Converged.Disabled and checked.default.chromium.png|240795| Changed|
|Switch Converged.Disabled and unchecked.default.chromium.png|240815| Changed|
|Switch Converged.Enabled and checked - RTL.default.chromium.png|240350| Changed|
|Switch Converged.Enabled and unchecked - High Contrast.default.chromium.png|238015| Changed|
|Switch Converged.Disabled and unchecked - High Contrast.default.chromium.png|237846| Changed|
|Switch Converged.Enabled and unchecked - Dark Mode.default.chromium.png|237304| Changed|
|Switch Converged.Enabled and checked.default.chromium.png|240349| Changed|
|Switch Converged.Enabled and unchecked - RTL.default.chromium.png|240549| Changed|
|Switch Converged.Enabled and checked - High Contrast.default.chromium.png|238085| Changed|
|Switch Converged.Enabled and unchecked.default.chromium.png|240548| Changed|
|Switch Converged.Enabled and checked - Dark Mode.default.chromium.png|237388| Changed|
|Switch Converged.With label above - RTL.default.chromium.png|240147| Changed|
|Switch Converged.Required.default.chromium.png|240824| Changed|
|Switch Converged.With label after.default.chromium.png|240548| Changed|
|Switch Converged.With label above.default.chromium.png|240140| Changed|
|Switch Converged.With label after - RTL.default.chromium.png|240549| Changed|
|Switch Converged.With label before - RTL.default.chromium.png|240548| Changed|
|Switch Converged.With label wrapping.default.chromium.png|260130| Changed|
|Switch Converged.With label before.default.chromium.png|240549| Changed|
|Switch Converged.Disabled and checked - Dark Mode.hover.chromium.png|0| Removed|
|Switch Converged.Disabled and checked - Dark Mode.pressed.chromium.png|0| Removed|
|Switch Converged.Disabled and checked - High Contrast.hover.chromium.png|0| Removed|
|Switch Converged.Disabled and checked - High Contrast.pressed.chromium.png|0| Removed|
|Switch Converged.Disabled and checked.hover.chromium.png|0| Removed|
|Switch Converged.Disabled and checked.pressed.chromium.png|0| Removed|
|Switch Converged.Disabled and unchecked - Dark Mode.hover.chromium.png|0| Removed|
|Switch Converged.Disabled and unchecked - Dark Mode.pressed.chromium.png|0| Removed|
|Switch Converged.Disabled and unchecked - High Contrast.hover.chromium.png|0| Removed|
|Switch Converged.Disabled and unchecked - High Contrast.pressed.chromium.png|0| Removed|
|Switch Converged.Disabled and unchecked.hover.chromium.png|0| Removed|
|Switch Converged.Disabled and unchecked.pressed.chromium.png|0| Removed|
|Switch Converged.Enabled and checked - Dark Mode.hover.chromium.png|0| Removed|
|Switch Converged.Enabled and checked - Dark Mode.pressed.chromium.png|0| Removed|
|Switch Converged.Enabled and checked - High Contrast.hover.chromium.png|0| Removed|
|Switch Converged.Enabled and checked - High Contrast.pressed.chromium.png|0| Removed|
|Switch Converged.Enabled and checked - RTL.hover.chromium.png|0| Removed|
|Switch Converged.Enabled and checked - RTL.pressed.chromium.png|0| Removed|
|Switch Converged.Enabled and checked.hover.chromium.png|0| Removed|
|Switch Converged.Enabled and checked.pressed.chromium.png|0| Removed|
|Switch Converged.Enabled and unchecked - Dark Mode.hover.chromium.png|0| Removed|
|Switch Converged.Enabled and unchecked - Dark Mode.pressed.chromium.png|0| Removed|
|Switch Converged.Enabled and unchecked - High Contrast.hover.chromium.png|0| Removed|
|Switch Converged.Enabled and unchecked - High Contrast.pressed.chromium.png|0| Removed|
|Switch Converged.Enabled and unchecked - RTL.hover.chromium.png|0| Removed|
|Switch Converged.Enabled and unchecked - RTL.pressed.chromium.png|0| Removed|
|Switch Converged.Enabled and unchecked.hover.chromium.png|0| Removed|
|Switch Converged.Enabled and unchecked.pressed.chromium.png|0| Removed|
|Switch Converged.Required.hover.chromium.png|0| Removed|
|Switch Converged.Required.pressed.chromium.png|0| Removed|
|Switch Converged.With label above - RTL.hover.chromium.png|0| Removed|
|Switch Converged.With label above - RTL.pressed.chromium.png|0| Removed|
|Switch Converged.With label above.hover.chromium.png|0| Removed|
|Switch Converged.With label above.pressed.chromium.png|0| Removed|
|Switch Converged.With label after - RTL.hover.chromium.png|0| Removed|
|Switch Converged.With label after - RTL.pressed.chromium.png|0| Removed|
|Switch Converged.With label after.hover.chromium.png|0| Removed|
|Switch Converged.With label after.pressed.chromium.png|0| Removed|
|Switch Converged.With label before - RTL.hover.chromium.png|0| Removed|
|Switch Converged.With label before - RTL.pressed.chromium.png|0| Removed|
|Switch Converged.With label before.hover.chromium.png|0| Removed|
|Switch Converged.With label before.pressed.chromium.png|0| Removed|
|Switch Converged.With label wrapping.hover.chromium.png|0| Removed|
|Switch Converged.With label wrapping.pressed.chromium.png|0| Removed|

</details>


  <details><summary><b>TabList and Tab Converged</b> 96 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TabList and Tab Converged.Default - Dark Mode.default.chromium.png|1545358| Changed|
|TabList and Tab Converged.Default - High Contrast.default.chromium.png|1557069| Changed|
|TabList and Tab Converged.Large size.default.chromium.png|1607384| Changed|
|TabList and Tab Converged.Subtle appearance - Dark Mode.default.chromium.png|1545358| Changed|
|TabList and Tab Converged.Default.default.chromium.png|1607458| Changed|
|TabList and Tab Converged.Large size - Dark Mode.default.chromium.png|1531552| Changed|
|TabList and Tab Converged.Default - RTL.default.chromium.png|1607458| Changed|
|TabList and Tab Converged.Small size - High Contrast.default.chromium.png|1573604| Changed|
|TabList and Tab Converged.Large size - High Contrast.default.chromium.png|1545857| Changed|
|TabList and Tab Converged.Small size - Dark Mode.default.chromium.png|1564935| Changed|
|TabList and Tab Converged.Small size.default.chromium.png|1607463| Changed|
|TabList and Tab Converged.Subtle appearance.default.chromium.png|1607458| Changed|
|TabList and Tab Converged.Subtle appearance - High Contrast.default.chromium.png|1557069| Changed|
|TabList and Tab Converged.Tab Selected (default) - Dark Mode.default.chromium.png|1545499| Changed|
|TabList and Tab Converged.Tab Selected (default).default.chromium.png|1607287| Changed|
|TabList and Tab Converged.Tab Selected (default) - High Contrast.default.chromium.png|1557180| Changed|
|TabList and Tab Converged.Vertical and large size - Dark Mode.default.chromium.png|1438816| Changed|
|TabList and Tab Converged.Tab Selected - Dark Mode.default.chromium.png|1545499| Changed|
|TabList and Tab Converged.Tab Selected - High Contrast.default.chromium.png|1557180| Changed|
|TabList and Tab Converged.Tab Selected.default.chromium.png|1607287| Changed|
|TabList and Tab Converged.Vertical and large size - High Contrast.default.chromium.png|1447958| Changed|
|TabList and Tab Converged.Vertical and large size.default.chromium.png|1607407| Changed|
|TabList and Tab Converged.Vertical and small size - High Contrast.default.chromium.png|1521309| Changed|
|TabList and Tab Converged.Vertical and small size.default.chromium.png|1607463| Changed|
|TabList and Tab Converged.Vertical and small size - Dark Mode.default.chromium.png|1502427| Changed|
|TabList and Tab Converged.Vertical.default.chromium.png|1607506| Changed|
|TabList and Tab Converged.With icon - RTL.default.chromium.png|1607506| Changed|
|TabList and Tab Converged.With icon and vertical - RTL.default.chromium.png|1607509| Changed|
|TabList and Tab Converged.With icon and vertical.default.chromium.png|1607566| Changed|
|TabList and Tab Converged.With icon only and vertical.default.chromium.png|1607753| Changed|
|TabList and Tab Converged.With icon only.default.chromium.png|1607752| Changed|
|TabList and Tab Converged.With icon.default.chromium.png|1607506| Changed|
|TabList and Tab Converged.Default - Dark Mode.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Default - Dark Mode.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Default - High Contrast.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Default - High Contrast.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Default - RTL.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Default - RTL.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Default.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Default.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Large size - Dark Mode.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Large size - Dark Mode.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Large size - High Contrast.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Large size - High Contrast.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Large size.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Large size.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Small size - Dark Mode.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Small size - Dark Mode.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Small size - High Contrast.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Small size - High Contrast.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Small size.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Small size.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Subtle appearance - Dark Mode.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Subtle appearance - Dark Mode.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Subtle appearance - High Contrast.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Subtle appearance - High Contrast.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Subtle appearance.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Subtle appearance.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Tab Selected (default) - Dark Mode.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Tab Selected (default) - Dark Mode.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Tab Selected (default) - High Contrast.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Tab Selected (default) - High Contrast.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Tab Selected (default).hover.chromium.png|0| Removed|
|TabList and Tab Converged.Tab Selected (default).pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Tab Selected - Dark Mode.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Tab Selected - Dark Mode.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Tab Selected - High Contrast.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Tab Selected - High Contrast.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Tab Selected.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Tab Selected.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Vertical and large size - Dark Mode.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Vertical and large size - Dark Mode.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Vertical and large size - High Contrast.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Vertical and large size - High Contrast.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Vertical and large size.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Vertical and large size.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Vertical and small size - Dark Mode.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Vertical and small size - Dark Mode.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Vertical and small size - High Contrast.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Vertical and small size - High Contrast.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Vertical and small size.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Vertical and small size.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Vertical.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Vertical.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.With icon - RTL.hover.chromium.png|0| Removed|
|TabList and Tab Converged.With icon - RTL.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.With icon and vertical - RTL.hover.chromium.png|0| Removed|
|TabList and Tab Converged.With icon and vertical - RTL.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.With icon and vertical.hover.chromium.png|0| Removed|
|TabList and Tab Converged.With icon and vertical.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.With icon only and vertical.hover.chromium.png|0| Removed|
|TabList and Tab Converged.With icon only and vertical.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.With icon only.hover.chromium.png|0| Removed|
|TabList and Tab Converged.With icon only.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.With icon.hover.chromium.png|0| Removed|
|TabList and Tab Converged.With icon.pressed.chromium.png|0| Removed|

</details>


  <details><summary><b>Table flex - subtle selection</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table flex - subtle selection.rest.chromium.png|1600438| Changed|
|Table flex - subtle selection.rest.hover unselected row.chromium.png|0| Removed|

</details>


  <details><summary><b>Table layout flex</b> 44 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout flex.multiselect (checked) - brand - High Contrast.chromium.png|1295235| Changed|
|Table layout flex.keyboard column resizing style.chromium.png|1596666| Changed|
|Table layout flex.multiselect (checked) - brand - RTL.chromium.png|1583470| Changed|
|Table layout flex.multiselect (checked) - brand - Dark Mode.chromium.png|1294746| Changed|
|Table layout flex.multiselect (checked) - neutral - High Contrast.chromium.png|1460964| Changed|
|Table layout flex.multiselect (mixed) - brand - Dark Mode.chromium.png|1297081| Changed|
|Table layout flex.multiselect (checked) - neutral - RTL.chromium.png|1575498| Changed|
|Table layout flex.multiselect (mixed) - brand - High Contrast.chromium.png|1296272| Changed|
|Table layout flex.multiselect (checked) - brand.chromium.png|1583355| Changed|
|Table layout flex.multiselect (mixed) - brand.chromium.png|1591839| Changed|
|Table layout flex.multiselect (mixed) - neutral - Dark Mode.chromium.png|1308264| Changed|
|Table layout flex.multiselect (checked) - neutral - Dark Mode.chromium.png|1315455| Changed|
|Table layout flex.multiselect (mixed) - neutral - RTL.chromium.png|1587954| Changed|
|Table layout flex.multiselect (mixed) - brand - RTL.chromium.png|1591953| Changed|
|Table layout flex.multiselect (mixed) - neutral - High Contrast.chromium.png|1385585| Changed|
|Table layout flex.multiselect (checked) - neutral.chromium.png|1575380| Changed|
|Table layout flex.multiselect (mixed) - neutral.chromium.png|1587837| Changed|
|Table layout flex.multiselect.chromium.png|1600337| Changed|
|Table layout flex.multiselect - Dark Mode.chromium.png|1303295| Changed|
|Table layout flex.multiselect - High Contrast.chromium.png|1297274| Changed|
|Table layout flex.multiselect - RTL.chromium.png|1600447| Changed|
|Table layout flex.primary cell.chromium.png|1599428| Changed|
|Table layout flex.single select (checked) - brand.chromium.png|1596252| Changed|
|Table layout flex.single select (checked) - Dark Mode.chromium.png|1303268| Changed|
|Table layout flex.single select (checked) - High Contrast.chromium.png|1297185| Changed|
|Table layout flex.single select (checked) - brand - Dark Mode.chromium.png|1299878| Changed|
|Table layout flex.single select (checked) - RTL.chromium.png|1600472| Changed|
|Table layout flex.single select (checked) - neutral - Dark Mode.chromium.png|1305708| Changed|
|Table layout flex.single select (checked).chromium.png|1600361| Changed|
|Table layout flex.single select (checked) - brand - High Contrast.chromium.png|1296589| Changed|
|Table layout flex.single select (checked) - neutral - High Contrast.chromium.png|1342199| Changed|
|Table layout flex.single select (checked) - neutral.chromium.png|1594251| Changed|
|Table layout flex.single select (checked) - brand - RTL.chromium.png|1596362| Changed|
|Table layout flex.single select - Dark Mode.chromium.png|1303243| Changed|
|Table layout flex.single select.chromium.png|1600409| Changed|
|Table layout flex.single select (checked) - neutral - RTL.chromium.png|1594361| Changed|
|Table layout flex.size - medium - Dark Mode.chromium.png|1303055| Changed|
|Table layout flex.size - extra small.chromium.png|1603339| Changed|
|Table layout flex.single select - High Contrast.chromium.png|1297171| Changed|
|Table layout flex.single select - RTL.chromium.png|1600519| Changed|
|Table layout flex.size - medium - High Contrast.chromium.png|1296910| Changed|
|Table layout flex.size - medium - RTL.chromium.png|1600586| Changed|
|Table layout flex.size - medium.chromium.png|1600573| Changed|
|Table layout flex.size - small.chromium.png|1600868| Changed|

</details>


  <details><summary><b>Table layout flex - cell actions</b> 18 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout flex - cell actions.always visible - Dark Mode.chromium.png|1303168| Changed|
|Table layout flex - cell actions.always visible - RTL.chromium.png|1600408| Changed|
|Table layout flex - cell actions.always visible - High Contrast.chromium.png|1297052| Changed|
|Table layout flex - cell actions.always visible.chromium.png|1600426| Changed|
|Table layout flex - cell actions.in header cell.chromium.png|1600573| Changed|
|Table layout flex - cell actions.default - RTL.chromium.png|1600586| Changed|
|Table layout flex - cell actions.default.chromium.png|1600573| Changed|
|Table layout flex - cell actions.default - Dark Mode.chromium.png|1303055| Changed|
|Table layout flex - cell actions.default - High Contrast.chromium.png|1296910| Changed|
|Table layout flex - cell actions.always visible - Dark Mode.hover row.chromium.png|0| Removed|
|Table layout flex - cell actions.always visible - High Contrast.hover row.chromium.png|0| Removed|
|Table layout flex - cell actions.always visible - RTL.hover row.chromium.png|0| Removed|
|Table layout flex - cell actions.always visible.hover row.chromium.png|0| Removed|
|Table layout flex - cell actions.default - Dark Mode.hover row.chromium.png|0| Removed|
|Table layout flex - cell actions.default - High Contrast.hover row.chromium.png|0| Removed|
|Table layout flex - cell actions.default - RTL.hover row.chromium.png|0| Removed|
|Table layout flex - cell actions.default.hover row.chromium.png|0| Removed|
|Table layout flex - cell actions.in header cell.hover row.chromium.png|0| Removed|

</details>


  <details><summary><b>Table layout flex - headers</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout flex - headers.sortable - High Contrast.chromium.png|1296919| Changed|
|Table layout flex - headers.sortable - Dark Mode.chromium.png|1303096| Changed|
|Table layout flex - headers.sortable.chromium.png|1600534| Changed|
|Table layout flex - headers.sortable - Dark Mode.hover header.chromium.png|0| Removed|
|Table layout flex - headers.sortable - Dark Mode.press header.chromium.png|0| Removed|
|Table layout flex - headers.sortable - High Contrast.hover header.chromium.png|0| Removed|
|Table layout flex - headers.sortable - High Contrast.press header.chromium.png|0| Removed|
|Table layout flex - headers.sortable.hover header.chromium.png|0| Removed|
|Table layout flex - headers.sortable.press header.chromium.png|0| Removed|

</details>


  <details><summary><b>Table layout flex - truncate</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout flex - truncate.false.chromium.png|1601603| Changed|
|Table layout flex - truncate.default (disabled).chromium.png|1601603| Changed|
|Table layout flex - truncate.true.chromium.png|1604335| Changed|

</details>


  <details><summary><b>Table layout table</b> 44 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout table.keyboard column resizing style.chromium.png|1596427| Changed|
|Table layout table.multiselect (checked) - brand - RTL.chromium.png|1583859| Changed|
|Table layout table.multiselect (checked) - neutral - Dark Mode.chromium.png|1315455| Changed|
|Table layout table.multiselect (checked) - brand - High Contrast.chromium.png|1295235| Changed|
|Table layout table.multiselect (checked) - neutral - RTL.chromium.png|1575498| Changed|
|Table layout table.multiselect (checked) - brand.chromium.png|1583744| Changed|
|Table layout table.multiselect (checked) - brand - Dark Mode.chromium.png|1295023| Changed|
|Table layout table.multiselect (mixed) - neutral.chromium.png|1587837| Changed|
|Table layout table.multiselect (checked) - neutral - High Contrast.chromium.png|1460964| Changed|
|Table layout table.multiselect (mixed) - neutral - RTL.chromium.png|1587954| Changed|
|Table layout table.multiselect (mixed) - brand - Dark Mode.chromium.png|1297204| Changed|
|Table layout table.multiselect (checked) - neutral.chromium.png|1575380| Changed|
|Table layout table.multiselect (mixed) - brand - High Contrast.chromium.png|1296272| Changed|
|Table layout table.multiselect (mixed) - brand - RTL.chromium.png|1592147| Changed|
|Table layout table.multiselect (mixed) - neutral - Dark Mode.chromium.png|1308264| Changed|
|Table layout table.multiselect - High Contrast.chromium.png|1297274| Changed|
|Table layout table.multiselect (mixed) - brand.chromium.png|1592033| Changed|
|Table layout table.multiselect - Dark Mode.chromium.png|1303295| Changed|
|Table layout table.multiselect (mixed) - neutral - High Contrast.chromium.png|1385585| Changed|
|Table layout table.multiselect - RTL.chromium.png|1600447| Changed|
|Table layout table.multiselect.chromium.png|1600337| Changed|
|Table layout table.primary cell.chromium.png|1599422| Changed|
|Table layout table.single select (checked) - High Contrast.chromium.png|1297185| Changed|
|Table layout table.single select (checked) - Dark Mode.chromium.png|1303268| Changed|
|Table layout table.single select (checked) - brand - Dark Mode.chromium.png|1299960| Changed|
|Table layout table.single select (checked) - RTL.chromium.png|1600472| Changed|
|Table layout table.single select (checked) - brand - High Contrast.chromium.png|1296589| Changed|
|Table layout table.single select (checked) - neutral - RTL.chromium.png|1594361| Changed|
|Table layout table.single select (checked).chromium.png|1600361| Changed|
|Table layout table.single select (checked) - neutral - Dark Mode.chromium.png|1305708| Changed|
|Table layout table.single select (checked) - neutral - High Contrast.chromium.png|1342199| Changed|
|Table layout table.single select (checked) - brand.chromium.png|1596350| Changed|
|Table layout table.single select (checked) - brand - RTL.chromium.png|1596459| Changed|
|Table layout table.single select (checked) - neutral.chromium.png|1594251| Changed|
|Table layout table.size - medium - High Contrast.chromium.png|1296910| Changed|
|Table layout table.single select - High Contrast.chromium.png|1297171| Changed|
|Table layout table.single select - Dark Mode.chromium.png|1303243| Changed|
|Table layout table.single select.chromium.png|1600409| Changed|
|Table layout table.size - small.chromium.png|1600868| Changed|
|Table layout table.single select - RTL.chromium.png|1600519| Changed|
|Table layout table.size - medium - Dark Mode.chromium.png|1303055| Changed|
|Table layout table.size - medium.chromium.png|1600573| Changed|
|Table layout table.size - extra small.chromium.png|1603339| Changed|
|Table layout table.size - medium - RTL.chromium.png|1600586| Changed|

</details>


  <details><summary><b>Table layout table - cell actions</b> 18 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout table - cell actions.always visible - Dark Mode.chromium.png|1303168| Changed|
|Table layout table - cell actions.default - RTL.chromium.png|1600586| Changed|
|Table layout table - cell actions.always visible - High Contrast.chromium.png|1297052| Changed|
|Table layout table - cell actions.in header cell.chromium.png|1600573| Changed|
|Table layout table - cell actions.default - High Contrast.chromium.png|1296910| Changed|
|Table layout table - cell actions.always visible - RTL.chromium.png|1600409| Changed|
|Table layout table - cell actions.always visible.chromium.png|1600426| Changed|
|Table layout table - cell actions.default - Dark Mode.chromium.png|1303055| Changed|
|Table layout table - cell actions.default.chromium.png|1600573| Changed|
|Table layout table - cell actions.always visible - Dark Mode.hover row.chromium.png|0| Removed|
|Table layout table - cell actions.always visible - High Contrast.hover row.chromium.png|0| Removed|
|Table layout table - cell actions.always visible - RTL.hover row.chromium.png|0| Removed|
|Table layout table - cell actions.always visible.hover row.chromium.png|0| Removed|
|Table layout table - cell actions.default - Dark Mode.hover row.chromium.png|0| Removed|
|Table layout table - cell actions.default - High Contrast.hover row.chromium.png|0| Removed|
|Table layout table - cell actions.default - RTL.hover row.chromium.png|0| Removed|
|Table layout table - cell actions.default.hover row.chromium.png|0| Removed|
|Table layout table - cell actions.in header cell.hover row.chromium.png|0| Removed|

</details>


  <details><summary><b>Table layout table - headers</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout table - headers.sortable - Dark Mode.chromium.png|1303096| Changed|
|Table layout table - headers.sortable - High Contrast.chromium.png|1296919| Changed|
|Table layout table - headers.sortable.chromium.png|1600534| Changed|
|Table layout table - headers.sortable - Dark Mode.hover header.chromium.png|0| Removed|
|Table layout table - headers.sortable - Dark Mode.press header.chromium.png|0| Removed|
|Table layout table - headers.sortable - High Contrast.hover header.chromium.png|0| Removed|
|Table layout table - headers.sortable - High Contrast.press header.chromium.png|0| Removed|
|Table layout table - headers.sortable.hover header.chromium.png|0| Removed|
|Table layout table - headers.sortable.press header.chromium.png|0| Removed|

</details>


  <details><summary><b>Table layout table - truncate</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout table - truncate.false.chromium.png|1601603| Changed|
|Table layout table - truncate.default (disabled).chromium.png|1601603| Changed|
|Table layout table - truncate.true.chromium.png|1604335| Changed|

</details>


  <details><summary><b>Table table - subtle selection</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table table - subtle selection.rest.chromium.png|1600438| Changed|
|Table table - subtle selection.rest.hover unselected row.chromium.png|0| Removed|

</details>


  <details><summary><b>Textarea Converged</b> 28 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Textarea Converged.Appearance- filled-lighter.default.chromium.png|253392| Changed|
|Textarea Converged.Appearance- outline (default).default.chromium.png|248108| Changed|
|Textarea Converged.Appearance- filled-darker.default.chromium.png|247853| Changed|
|Textarea Converged.Disabled + filled.default.chromium.png|248241| Changed|
|Textarea Converged.Disabled.default.chromium.png|248241| Changed|
|Textarea Converged.Disabled + placeholder.default.chromium.png|248206| Changed|
|Textarea Converged.Invalid- filled-darker.default.chromium.png|247570| Changed|
|Textarea Converged.Size- medium.default.chromium.png|248108| Changed|
|Textarea Converged.Invalid- filled-lighter.default.chromium.png|253080| Changed|
|Textarea Converged.With value.default.chromium.png|248147| Changed|
|Textarea Converged.Invalid- outline.default.chromium.png|247981| Changed|
|Textarea Converged.Size- small.default.chromium.png|245489| Changed|
|Textarea Converged.With appearance override.default.chromium.png|263896| Changed|
|Textarea Converged.Size- large.default.chromium.png|251183| Changed|
|Textarea Converged.Appearance- filled-darker.focused.chromium.png|0| Removed|
|Textarea Converged.Appearance- filled-darker.hover.chromium.png|0| Removed|
|Textarea Converged.Appearance- filled-lighter.focused.chromium.png|0| Removed|
|Textarea Converged.Appearance- filled-lighter.hover.chromium.png|0| Removed|
|Textarea Converged.Appearance- outline (default).focused.chromium.png|0| Removed|
|Textarea Converged.Appearance- outline (default).hover.chromium.png|0| Removed|
|Textarea Converged.Invalid- filled-darker.focused.chromium.png|0| Removed|
|Textarea Converged.Invalid- filled-darker.hover.chromium.png|0| Removed|
|Textarea Converged.Invalid- filled-lighter.focused.chromium.png|0| Removed|
|Textarea Converged.Invalid- filled-lighter.hover.chromium.png|0| Removed|
|Textarea Converged.Invalid- outline.focused.chromium.png|0| Removed|
|Textarea Converged.Invalid- outline.hover.chromium.png|0| Removed|
|Textarea Converged.With value.focused.chromium.png|0| Removed|
|Textarea Converged.With value.hover.chromium.png|0| Removed|

</details>


  <details><summary><b>Toast</b> 42 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Toast.Full Toast - Dark Mode.chromium.png|1583317| Changed|
|Toast.Full Toast - High Contrast.chromium.png|1588921| Changed|
|Toast.Full Toast - RTL.chromium.png|1606962| Changed|
|Toast.Full Toast Inverted - Dark Mode.chromium.png|1583317| Changed|
|Toast.Full Toast.chromium.png|1606968| Changed|
|Toast.Full Toast Inverted - High Contrast.chromium.png|1588921| Changed|
|Toast.Title Only - Dark Mode.chromium.png|1583317| Changed|
|Toast.Full Toast Inverted.chromium.png|1606968| Changed|
|Toast.Title Only - RTL.chromium.png|1606962| Changed|
|Toast.Title Only - High Contrast.chromium.png|1588921| Changed|
|Toast.Title Only Inverted.chromium.png|1606968| Changed|
|Toast.Title Only Inverted - Dark Mode.chromium.png|1583317| Changed|
|Toast.Without Subtitle - Dark Mode.chromium.png|1583317| Changed|
|Toast.Without Subtitle Inverted - Dark Mode.chromium.png|1583317| Changed|
|Toast.Title Only.chromium.png|1606968| Changed|
|Toast.Without Subtitle - High Contrast.chromium.png|1588921| Changed|
|Toast.Without Subtitle Inverted.chromium.png|1606968| Changed|
|Toast.Title Only Inverted - High Contrast.chromium.png|1588921| Changed|
|Toast.Without Subtitle - RTL.chromium.png|1606962| Changed|
|Toast.Without Subtitle.chromium.png|1606968| Changed|
|Toast.Without Subtitle Inverted - High Contrast.chromium.png|1588921| Changed|
|Toast.Full Toast - Dark Mode.Toast visible.chromium.png|0| Removed|
|Toast.Full Toast - High Contrast.Toast visible.chromium.png|0| Removed|
|Toast.Full Toast - RTL.Toast visible.chromium.png|0| Removed|
|Toast.Full Toast Inverted - Dark Mode.Toast visible.chromium.png|0| Removed|
|Toast.Full Toast Inverted - High Contrast.Toast visible.chromium.png|0| Removed|
|Toast.Full Toast Inverted.Toast visible.chromium.png|0| Removed|
|Toast.Full Toast.Toast visible.chromium.png|0| Removed|
|Toast.Title Only - Dark Mode.Toast visible.chromium.png|0| Removed|
|Toast.Title Only - High Contrast.Toast visible.chromium.png|0| Removed|
|Toast.Title Only - RTL.Toast visible.chromium.png|0| Removed|
|Toast.Title Only Inverted - Dark Mode.Toast visible.chromium.png|0| Removed|
|Toast.Title Only Inverted - High Contrast.Toast visible.chromium.png|0| Removed|
|Toast.Title Only Inverted.Toast visible.chromium.png|0| Removed|
|Toast.Title Only.Toast visible.chromium.png|0| Removed|
|Toast.Without Subtitle - Dark Mode.Toast visible.chromium.png|0| Removed|
|Toast.Without Subtitle - High Contrast.Toast visible.chromium.png|0| Removed|
|Toast.Without Subtitle - RTL.Toast visible.chromium.png|0| Removed|
|Toast.Without Subtitle Inverted - Dark Mode.Toast visible.chromium.png|0| Removed|
|Toast.Without Subtitle Inverted - High Contrast.Toast visible.chromium.png|0| Removed|
|Toast.Without Subtitle Inverted.Toast visible.chromium.png|0| Removed|
|Toast.Without Subtitle.Toast visible.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>